commands/add/index.js

```diff
@@ -36,6 +36,13 @@ class AddCommand extends Command {
   }
 
   initialize() {
+    if (this.options.npmClient === "pnpm") {
+      throw new ValidationError(
+        "EPNPMNOTSUPPORTED",
+        "The 'add' command is not supported when using `pnpm` workspaces. Use `pnpm` directly to add dependencies to packages: https://pnpm.io/cli/add."
```

Comment:
I think it's cleaner to not have the period after the URL (and maybe it would mess with terminals trying to make it clickable?)

```suggestion
        "The 'add' command is not supported when using `pnpm` workspaces. Use `pnpm` directly to add dependencies to packages: https://pnpm.io/cli/add"
```